src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -865,6 +865,36 @@ public void AnalyzeCommand_EncodedPaths()
             sarifLog.Runs[0].Results.Count.Should().Be(1);
         }
 
+        [Fact]
+        public void AnalyzeCommand_InvalidPaths()
+        {
+            var logger = new MemoryStreamSarifLogger(dataToInsert: OptionallyEmittedData.Hashes);
```

Comment:
Subdirectory test.  Ie: Direct%20dory\filename.txt doesn't confuse the encoder/decoder.

Reply:
The test case above this one is of the style 'Direct%20dory\filename.txt'.